config/kubernetes-sigs/org.yaml

```diff
@@ -18,6 +18,7 @@ admins:
 members:
 - aaroniscode
 - aaronlevy
+- abhiraut
```

Comment:
@nikhita seems like I received the invitation to join to repo while I was on PTO and the invitation has now expired. How do I receive another invitation? Thanks